nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -241,6 +242,151 @@ def _format_arg(self, name, trait_spec, value):
                 return f"-{trait_spec.argstr}"
         return super()._format_arg(name, trait_spec, value)
 
+
+class DWIPreprocInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True, argstr="%s", position=-10, mandatory=True, desc="input DWI image"
+    )
+    out_file = File(
+        "preproc.mif",
+        argstr="%s",
+        mandatory=True,
+        position=-9,
+        usedefault=True,
+        desc="output file after preprocessing",
+    )
+    rpe_options = traits.Enum(
+        "-rpe_none",
+        "-rpe_pair",
+        "-rpe_all",
+        "-rpe_header",
+        argstr="%s",
```

Comment:
These could be a bit more user-friendly:

```suggestion
        "none",
        "pair",
        "all",
        "header",
        argstr="-rpe_%s",
```